istio-control/istio-discovery/values.yaml

```diff
@@ -105,7 +105,7 @@ pilot:
 
     v1:
       # Will define mixerCheckServer and mixerReportServer
-      enabled: false
+      enabled: true
```

Comment:
From an API perspective, we just want to say `v2.enabled: true`  at present part of v1 is implemented by `v1` but this is implementation detail.

This should still be `false`

Reply:
done, PTAL